pymc_experimental/model_builder.py

```diff
@@ -400,8 +423,8 @@ def load(cls, fname: str):
 
     def fit(
         self,
-        X: Union[np.ndarray, pd.DataFrame, pd.Series],
-        y: Union[np.ndarray, pd.Series],
+        X: pd.DataFrame,
+        y: pd.Series,
```

Comment:
Can we make `y` be optional.

Reply:
The issue is that if we don't do that it will limit the models we can fit this on, as not all actually have an X and a y, for example, the change-point model only has a single time-series.